docs/_source/guides/llms/examples/curating-feedback-instructiondataset.ipynb

```diff
@@ -45,7 +45,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%pip install argilla datasets pandas httpx plotly -qqq"
+    "%pip install argilla==1.11.0 datasets pandas plotly -qqq"
```

Comment:
This version will be outdated quickly. Maybe just an `-U` to keep packages updated?

Reply:
For me make sense to align with the rest of the tutorials. If this is the way to install argilla in other tutorials, let's keep it as is.

Reply:
It was internally agreed that we should use `--upgrade` instead, and we'll review the rest of the tutorials iteratively in upcoming releases.